.circleci/config.templ.yml

```diff
@@ -1345,7 +1350,6 @@ jobs:
     # build documentation and store as an artifact
     executor: ddtrace_dev
     steps:
-      - setup_hatch
```

Comment:
To clarify, we're able to remove this but still run `hatch run ...` because it's already installed in `ddtrace_dev`?

Reply:
Yes, `ddtrace_dev` is an executor which uses testrunner docker image.